classes/Controller.php

```diff
@@ -0,0 +1,32 @@
+<?php 
+abstract class Controller{
+    protected $request;
+    protected $action;
+
+    public function __construct($request, $action){
+        $this->action = $action;
+        $this->request= $request;
+    }
+
+    public function executeAction(){
+        return $this->{$this->action}();
+    }
+
+    protected function returnView($viewmodel, $fullview){
+        $view = 'views/'.get_class($this).'/'.$this->action.'.php';
+        if($fullview){
```

Comment:
musisz sobie ogarnąć jakiś styl programowania (spacje, nowe linie itp)
najczęściej spotykane jest, że np po if, for, foreach, itp dajemy spacje czyli `if (warunek) { }` i pomiędzy okrągłym nawiasem a klamrowym też spacja
ale np jak piszesz funkcje, czy klasę to nie piszesz nawiasów klamrowych w jednej linii z nazwą klasy czy funkcji, czyli tak:
```
class Klasa
{
    public function test(): bool
    {
        if (1 === 2) {
            return true;
        }
        return false;
    }
}
```